pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-discord/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, pkgconfig, pidgin, json_glib }:
+
+stdenv.mkDerivation rec {
+  name = "pidgin-discord-${version}";
+  version = "0.9.2018.04.10.git.9a97886";
+
+  src = fetchFromGitHub {
+    owner = "EionRobb";
+    repo = "purple-discord";
+    rev = "9a97886d15a1f028de54b5e6fc54e784531063b0";
+    sha256 = "0dc344zh1v4yh9c8javcw5ylzwc1wpx0ih8bww8p8cjmhr8kcl32";
+  };
+
+  buildInputs = [ pkgconfig pidgin json_glib ];
+
+  makeFlags = [
+    "PLUGIN_DIR_PURPLE=/lib/pidgin/"
+    "DATA_ROOT_DIR_PURPLE=/share"
+    "DESTDIR=$(out)"
+  ];
+
+  installPhase = ''
+    install -Dm755 -t $out/lib/pidgin/ libdiscord.so
```

Comment:
Does not the default install phase work? You can overwrite the pkgconfig variables using `PKG_CONFIG_${package}_${variable}` (e.g. `PKG_CONFIG_PURPLE_PLUGINDIR = "lib/pidgin";`)